prow/lib.sh

```diff
@@ -112,3 +112,21 @@ function clone_cni() {
       cd "${TMP_DIR}" || return
   fi
 }
+
+function cni_run_daemon() {
+
+  echo 'Run the CNI daemon set'
+  ISTIO_CNI_HUB=${ISTIO_CNI_HUB:-gcr.io/istio-release}
+  ISTIO_CNI_TAG=${ISTIO_CNI_TAG:-master-latest-daily}
+
+  chartdir=$(pwd)/charts
+  mkdir "${chartdir}"
+  helm init --client-only
+  helm repo add istio.io https://storage.googleapis.com/istio-release/releases/1.1.0-snapshot.6/charts
```

Comment:
perhaps a nitpick here but can you use the daily releases to pick up bug fixes rather than using a snapshot?

Reply:
Sure I can change that - mostly wasn't aware of there being daily releases.